hedera-node/hedera-mono-service/src/main/resources/bootstrap.properties

```diff
@@ -127,6 +127,8 @@ scheduling.longTermEnabled=true
 scheduling.maxNumber=10_000_000
 scheduling.maxTxnPerSecond=100
 scheduling.maxExpirationFutureSeconds=5356800
```

Comment:
Curious why this value is ever-so-slightly more than 60 days?

Reply:
Yeah , not sure. It is 62 days.  🤔 Will change this back to 5184000 ?